modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/NestedEntityObjectEntry1toMRelatedModelsProviderImpl.java

```diff
@@ -18,52 +18,49 @@
 import com.liferay.object.model.ObjectDefinition;
 import com.liferay.object.model.ObjectEntry;
 import com.liferay.object.model.ObjectRelationship;
-import com.liferay.object.related.models.NestedEntityRelatedModelsProvider;
-import com.liferay.object.service.ObjectFieldLocalService;
 import com.liferay.portal.kernel.util.StringUtil;
 
 import java.util.Map;
 
+import javax.ws.rs.BadRequestException;
+
 /**
  * @author Sergio Jiménez del Coso
  */
 public class NestedEntityObjectEntry1toMRelatedModelsProviderImpl
-	implements NestedEntityRelatedModelsProvider {
+	extends BaseNestedEntityObjectEntryRelatedModelProviderImpl {
 
 	public NestedEntityObjectEntry1toMRelatedModelsProviderImpl(
-		ObjectDefinition objectDefinition,
-		ObjectFieldLocalService objectFieldLocalService) {
+		ObjectDefinition objectDefinition) {
 
-		_objectDefinition = objectDefinition;
-		_objectFieldLocalService = objectFieldLocalService;
+		super(objectDefinition);
 	}
 
 	@Override
 	public String getClassName() {
-		return _objectDefinition.getClassName();
+		return objectDefinition.getClassName();
```

Comment:
This method is exactly the same in both implementations, could it be moved to the base class?